### ISSUE TITLE
[BUG] data integrity suite passes when given a non-existing column to ignore

### ISSUE THREAD

**Issue by adfi** [deepchecks/deepchecks#2711]
**Describe the bug**
The Data Integrity suite passes when given a non-existing column to ignore.

**To Reproduce**
```
dummy_data = pd.DataFrame(
        {"COL1": [1] * 4, "COL2": [1, 2, 3, 4]}
)

dummy_ds = Dataset(dummy_data, cat_features=[])

suite = data_integrity(ignore_columns=['non_existing_column'])
suite_result = suite.run(dummy_ds)
suite_result.passed()
```

When not given any columns to ignore the suite doesn't pass.

**Expected behavior**
I expected the suite to not pass.

**Environment (please complete the following information):**
 - OS: Ubuntu
 - Python Version: 3.8.18
 - Deepchecks Version: 0.17.5

**Comment by adfi** [comment]
Ok I dived into it a little deeper and it seems this is perhaps excepted behaviour. Although that isn't really clear. 

If any columns are given that do not exist, the check throws an exception. The suite catches the exception and just doesn't run the check. I wasn't aware I had to also run `suite_result.get_not_ran_checks()` to find this out. I would've much rather found out which checks weren't being run from the logging or something.

So perhaps not a bug but maybe a request to improve the documentation?
